snap/snapcraft.yaml

```
@@ -0,0 +1,33 @@
+name: flamenco
+base: core22
+version: '1.0.0'
+summary: Package maintenance tool for Ubuntu Toolchains
+description: |
+  Flamenco is a CLI tool that helps toolchain developers manage many
+  different package versions and releases from a single debian
+  folder source tree.
+
+grade: stable
```

issue: The snap is still in development, therefore this can't be considered stable.

```suggestion
grade: devel
```

---

Sounds good to me. We'd only be able to push to latest/edge in that case, but I guess that's fine for as long as we're in development.